packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,524 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+- $p-color-theme-dark-notification-success-soft
+- $p-color-theme-dark-notification-warning
+- $p-color-theme-dark-notification-warning-soft
+- $p-color-theme-dark-notification-error
+- $p-color-theme-dark-notification-error-soft
+- $p-color-theme-dark-notification-neutral
+- $p-color-theme-dark-notification-neutral-soft
+- $p-color-theme-dark-state-hover
+- $p-color-theme-dark-state-active
+- $p-color-theme-dark-state-focus
+- $p-color-theme-dark-state-disabled
+
++ $pds-theme-dark-primary
++ $pds-theme-dark-background-base
++ $pds-theme-dark-background-surface
++ $pds-theme-dark-background-shading
++ $pds-theme-dark-contrast-low
++ $pds-theme-dark-contrast-medium
++ $pds-theme-dark-contrast-high
++ $pds-theme-dark-notification-success
++ $pds-theme-dark-notification-success-soft
++ $pds-theme-dark-notification-warning
++ $pds-theme-dark-notification-warning-soft
++ $pds-theme-dark-notification-error
++ $pds-theme-dark-notification-error-soft
++ $pds-theme-dark-notification-info
++ $pds-theme-dark-notification-info-soft
++ $pds-theme-dark-state-hover
++ $pds-theme-dark-state-active
++ $pds-theme-dark-state-focus
++ $pds-theme-dark-state-disabled
+```
+
+**Font**
+
+```diff
+- $p-font-family
++ $pds-font-family
+```
+
+```diff
+- $p-font-weight-regular
+- $p-font-weight-semibold
+- $p-font-weight-bold
+
++ $pds-font-weight-regular
++ $pds-font-weight-semi-bold
++ $pds-font-weight-bold
+```
+
+```diff
+- $p-font-size-x-small
+- $p-font-size-small
+- $p-font-size-medium
+- $p-font-size-large
+- $p-font-size-x-large
+
++ $pds-font-size-text-x-small
++ $pds-font-size-text-small
++ $pds-font-size-text-medium
++ $pds-font-size-text-large
++ $pds-font-size-text-x-large
+```
+
+**Spacings**
+
+With the upcoming v3 release, we will be introducing fluid spacers alongside our static spacers. These spacers will
+adjust in size based on a min and max value, providing a linear response to changes in browser width. Additionally,
+we've switched from using _rem_ to _px_ for our spacers to ensure they don't unnecessarily expand when the root font
+size is adjusted.
+
+To get an overview ot the new spacings have a look at [Spacing](https://designsystem.porsche.com/latest/styles/spacing).
+
+```diff
+- $p-spacing-4
+- $p-spacing-8
+- $p-spacing-16
+- $p-spacing-24
+- $p-spacing-32
+- $p-spacing-40
+- $p-spacing-48
+- $p-spacing-56
+- $p-spacing-64
+- $p-spacing-72
+- $p-spacing-80
+- $p-layout-x-small
+- $p-layout-small
+- $p-layout-medium
+- $p-layout-large
+- $p-layout-x-large
+- $p-layout-xx-large
+
++ $pds-spacing-static-x-small
++ $pds-spacing-static-small
++ $pds-spacing-static-medium
++ $pds-spacing-static-large
++ $pds-spacing-static-x-large
++ $pds-spacing-static-xx-large
++ $pds-spacing-fluid-x-small
++ $pds-spacing-fluid-small
++ $pds-spacing-fluid-medium
++ $pds-spacing-fluid-large
++ $pds-spacing-fluid-x-large
++ $pds-spacing-fluid-xx-large
+```
+
+#### Deprecated
+
+##### JS
+
+**Functions**
+
+- `pxToRem()` there is no replacement for the function.
```

Comment:
Maybe change the wording so that is an actual sentence or is there supposed to be a colon `:` after the function?